src/main.js

```diff
@@ -23,13 +26,27 @@ const app = new Vue({
   data: {
     user: null
   },
-  async mounted() {
-    try {
-      this.user = await getCurrentUser()
-    } catch (err) {
-      console.error(err.message)
+  computed: {
+    ...mapGetters(['showSearchResults'])
+  },
+  methods: {
+    ...mapActions(['setPreferredLanguage']),
+    async loadUser() {
+      try {
+        this.user = await getCurrentUser()
+      } catch (err) {
+        console.error(err.message)
+      }
+    },
+    initStateFromCookies() {
+      this.setPreferredLanguage(
+        readCookie(Cookie.preferredLanguage) || ''
```

Comment:
It looks like the cookie is being set for a particular url, so if you set a language on the data-api-specification page, and then go to the api-specification page, you don't see the language you set

Reply:
Thanks for catching this! Updating.